events_api.py

```diff
@@ -4,7 +4,7 @@
 
 def getEvents(lat, long):
   events_token = config('EVENTFUL_API_KEY')
-  events_request = requests.get('http://api.eventful.com/json/events/search?...&where='+lat+','+long+'&within=20&app_key='+events_token+'')
+  events_request = requests.get('http://api.eventful.com/json/events/search?...&where='+lat+','+long+'&within=20&app_key='+events_token+'&date=Today')
```

Comment:
Sending in the parameter for current day is a good change. We definitely don't want the possibility of it returning historical events.